src/docs/protocol/Architecture.md

```diff
@@ -6,25 +6,25 @@ lang: en-US
 
 ## TL;DR
 
-1. Due to its better composability, the modular design of blockchain has become a trend, and we have introduced this design into the construction of L2. 
-2. The architectural design of Morph can be simply divided into three parts: high-level consensus-execution-state verification, which correspond to different network functions. 3.
-3. At the same time, each part contains different components that can be combined arbitrarily to perform functions in different roles. Different roles also correspond to multiple components to maintain independence.
+1. Due to its improved composability, the modular design of blockchain has become a trend, and we have incorporated this design into the construction of Morphism.
+2. The architectural design of Morphism can be divided into three parts: high-level consensus, execution, and state verification, which correspond to different functions.
+3. Each part contains different components that can be combined in various ways to perform functions in different roles. Different roles also correspond to multiple components to maintain independence..
 
 ## Overview
 
+A general Layer 2 workflow is as follows:
 
+1. Users send transactions to sequencers.
+2. Sequencers receive the transactions, generate blocks, and execute them.
+3. Transactions are put into batches and submitted to Layer 1 contracts for data availability.
 
-A General Layer 2 workflow is like this:
-1. Users send transactions to sequencers
-2. Sequencer receives the transaction, generates block and excute block
-3. Transactions will be put into batches and submitted to Layer1 Contracts for data availability
+In addition to the basic workflow, Morphism aims to achieve a balance between decentralization, security, and scalability by implementing various measures, including:
 
-Besides the basic workflow, by implementing various measures, Morphism aims to achieve a balance between decentralization, security, and scalability, this includes:
-- Sequencers need to reach consensus with each other before executing transactions
-- Validator (Any user that runs L2 Node)can start a challenge that requires Sequencers to provide the zk Proof
-- Sequencers can call the zkProver to generate Validity Proof
+- Sequencers needing to reach consensus with each other before executing transactions.
```

Comment:
Sequencers need to reach a consensus with each other before executing transactions.